[ADD] estate: initialize the estate app

Faruk372742 opened the pull request:
Creating the manifest file.

Faruk372742 committed:
[ADD] estate: initialize the estate app

Creating the manifest file.

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/tutorials/pull/1061.png)](https://mergebot.odoo.com/odoo/tutorials/pull/1061)

Faruk372742 committed:
[ADD] estate: adding estate property table

Faruk372742 committed:
[ADD] estate: adding security for estate_property table

New ir.model.access file to adding all kind of accesses to the base.group_user and connecting csv file to manifest
